### PR TITLE
Add Udders

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add an Udders mutation"

#### Purpose of change
Add a new bovine mutation, Udders.

#### Describe the solution
Adds new mutation, Udders. With this, the player grows a large set of mammary glands around their lower abdomen, which blocks rigid armor and gets in the way, along with increasing their metabolism and thirst modifiers. However, once every twelve hours, they can be milked for dairy products.

#### Describe alternatives you've considered
Not doing this.

#### Testing
I tested everything, and it all works as I planned. However, I couldn't figure out how to have the milk spawn open the "place in container" prompt like other milking activities do, so I had to leave it as is.

#### Additional context
I couldn't figure out how to gate the mutation behind the characters sex either, so both men and women can have udders. If it is desired to be kept that way, I'd chalk it up to mutative shenanigans. Meanwhile, I'll do some searching myself for how it can be gated behind sex.

### CODE REVIEW

**Comment by Venera3** [comment]
Yeah, I'd really not do cow tits on players.

File: data/json/effect_on_condition.json [hunk 1]
```diff
@@ -1304,5 +1304,38 @@
...
+    "id": "EOC_milk_yourself",
+    "condition": {
+      "u_compare_time_since_var": "milk_regeneration",
+      "type": "timer",
+      "context": "bio_processies",
+      "op": ">=",
+      "time": "12 h"
+    },
+    "effect": [
+      { "u_spawn_item": "milk_raw", "count": 40 },
```
Comment (andrei8l) [inline data/json/effect_on_condition.json:1320]: This should cost at least as many calories as the produced milk (5280 Calories). 1.2 * base metabolism over 12 hours is just 1080 extra Calories.

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -1092,6 +1092,25 @@
...
+    "id": "UDDERS",
+    "name": { "str": "Udders" },
+    "points": 2,
+    "description": "You have grown a pair of udders on your lower abdomen.  They protrude greatly, and can get in the way of things, and increase your metabolism.  You can milk them, but be sure to do it over a bathtub to catch it all.",
+    "types": [ "SECRETION" ],
+    "category": [ "CATTLE" ],
+    "restricts_gear": [ "torso" ],
+    "remove_rigid": [ "torso" ],
+    "allow_soft_gear": true,
+    "metabolism_modifier": 1.2,
```
Comment (MNG-cataclysm) [inline data/json/mutations/mutations.json:1106]:
```suggestion
    "metabolism_modifier": 4.8
```

File: data/json/effect_on_condition.json (hunk 1 above)
Comment (MNG-cataclysm) [reply data/json/effect_on_condition.json:1320]: How does a multiplayer of 5 sound?

**Comment by esotericist** [comment]
yeah, this isn't happening. aside from the obvious issues venera3 touched on, there's also our usual "biological processes" issue.

firm no.

**Comment by Maleclypse** [comment]
I know you weren't here for the event years ago that I'm referencing but we have a hard line in game for representing biological sex and this would run afoul of that. I don't see any way for this not to run afoul of that line and so I'm closing it before you invest too much work into it.   This isn't a comment on you or what you bring but just a statement that biological sex is off limits for DDA.

**Comment by MNG-cataclysm** [comment]
> I know you weren't here for the event years ago that I'm referencing but we have a hard line in game for representing biological sex and this would run afoul of that. I don't see any way for this not to run afoul of that line and so I'm closing it before you invest too much work into it.   This isn't a comment on you or what you bring but just a statement that biological sex is off limits for DDA.

What event was it that happened years ago?

**Comment by Maleclypse** [comment]
> > I know you weren't here for the event years ago that I'm referencing but we have a hard line in game for representing biological sex and this would run afoul of that. I don't see any way for this not to run afoul of that line and so I'm closing it before you invest too much work into it.   This isn't a comment on you or what you bring but just a statement that biological sex is off limits for DDA.
> 
> What event was it that happened years ago?

The last time we made it clear that the game only tracks gender.

**Comment by Zireael07** [comment]
@Maleclypse How does a "hard no" work since we have dialogue with Alondo that's fairly clearly suggesting what's going on?

**Comment by esotericist** [comment]
> @Maleclypse How does a "hard no" work since we have dialogue with Alondo that's fairly clearly suggesting what's going on?

@Zireael07 i think you think you're replying to a different PR than you are.

**Comment by Zireael07** [comment]
I'm replying to this "we have a hard line for representing biological sex in game"

**Comment by Zireael07** [comment]
@esotericist Ah! Not a native speaker of English, I thought "biological sex" covered that Alondo scene.

(That other issue was locked almost immediately, so it was impossible to ask there, and I didn't want to create a third issue which would drag the comment from here as a starter...)

**Comment by esotericist** [comment]
okay, that's fair as a misunderstanding. if you _really_ want to engage with someone further on the matter, discord is an option; a lot of our discussions happen there. but as you may have gathered, it's a very touchy topic, so be careful in how you approach it.

for now, i'm going to go ahead and lock this because a) _this_ is pretty firmly resolved, b) closed PRs shouldn't be gathering traffic.

**Pull request closed without merging**